### ISSUE TITLE
getCurrentPosition sends 408 a lot of times

### ISSUE THREAD

**Issue by Ruzen1** [transistorsoft/react-native-background-geolocation#1325]
## Your Environment
* Plugin version:  ^3.9.0
* Platform: iOS or Android
* OS version: i0S 14.6, Android 11
* Device manufacturer / model: Apple/Google Pixel
* React Native version (`react-native -v`): 0.61.3
* Plugin config
```javascript <!-- Syntax-highlighting:  paste your code below -->
 BackgroundGeolocation.ready(
      {
        locationAuthorizationRequest: "Always",
        disableLocationAuthorizationAlert: true,
        desiredAccuracy: BackgroundGeolocation.DESIRED_ACCURACY_HIGH,
        stopTimeout: 10,
        debug: false,
        logLevel: BackgroundGeolocation.LOG_LEVEL_ERROR,
        stopOnTerminate: false,
        startOnBoot: true,
        autoSync: false,
        enableHeadless: true,
        motionTriggerDelay: 30000,
        preventSuspend: true 
      },
      async (state) => {
        if (!state.enabled) {
          const locationPermission = await Permission.checkPermissionLocation()
          if (locationPermission) {
            BackgroundGeolocation.start()
          }
        }
      }
    )
```

## Expected Behavior

We wish to reduce the 408 error code while using the getCurrentPosition which has timeout of 3 seconds. Is there any way to 
reduce the 408 errors? I have gone through 
https://github.com/transistorsoft/react-native-background-geolocation/issues/395#issuecomment-526593086 
and came to know that the problem is probably the location where the user is. But the thing is, it just fails for some time. If we keep trying at the same place, it takes the location perfectly.
## Actual Behavior

getCurrentPosition receives 408 a couple of times after waiting for like 3 seconds. But in the same location, if we keep trying for some time, it receives the location

## Context

We are trying to get the location using getCurrentPosition.

</details>

**Comment by christocracy** [comment]
It means timeout.  The device's location API failed to provide the plugin a location within locationTimeout and the plugin gave up.

**Comment by Ruzen1** [comment]
Thank you . Will try increasing the timeout time.

**Comment by tristanheilman** [comment]
I noticed that I run into this issue with only getting back 408s on location WHEN I navigate to
**Settings** -> **Location** -> **Location Services** and turn off
- _Google Location Accuracy_
- _Google Location History_
- _Google Location Sharing_

In my app code, I am attempting to fetch a new location inside the onHeartbeat listener. I have adjusted the location timeout from 5 -> 15 and no changes. It seems the plugin hits a timeout no matter what. Simply turning these settings back to **On** fixes the issue. 

My only concern is that many of our users are very dumb. They would need somebody holding their hand to help them make sure this is turned on. To my knowledge there is no permissions hook to navigate a user to this setting to help them turn it on quickly.

Any suggestions?
